### DISCUSSION TITLE
Namspace inside the values file

### DISCUSSION THREAD

**Discussion by novasolei** [sse-secure-systems/connaisseur#864] [Q&A]
Hello folks,
Is there an option to put the namespace inside the values.yaml instead of providing it during installation of connaiessuer? 
This way we can use different values.yaml for different environment without making any mistakes when running through command line. 
thanks

**Comment by xopham** [comment]
Hey @novasolei , currently there is no option to do so because that is a bit of a tricky topic and there has been considerable discussion on it elsewhere: https://github.com/helm/helm/issues/5465
One major question is how to handle the Helm `--namespace` and `--create-namespace` flags in that case. Not sure, how that would best be implemented.
